src/script.js

```diff
@@ -0,0 +1,28 @@
+// import modules
+import css from './modules/css.js';
+
+//  New methods of Element object
+Element.prototype.css = css;
+NodeList.prototype.css = css;
+
+
+// Testing
+const body = document.body;
+
+let element1 = document.querySelector('#one');
+let element2 = document.querySelector('#two');
+let element3 = document.querySelector('#three');
+
+function returnColor() {
```

Comment:
I think the callback function has to accept the computed style object. In your current case, you can't have access to the real value

Reply:
I'm sorry, I didn't understand this point. Do you mean returnColor() function? I have used it just to test if my method can call a function, and use a returned value.

Reply:
My bad. That's OK ) 👍